aimage/functions.py

```diff
@@ -51,11 +52,8 @@ async def generate_image(self,
         if await self._contains_blacklisted_word(guild, prompt):
             return self.send_response(context, content=":warning: Prompt contains blacklisted words!")
 
-        if lora:
-            prompt += lora  # loras are parsed out of the prompt beforehand
-
         payload = payload or {
-            "prompt": prompt,
+            "prompt": prompt + lora,  # loras are parsed out of the prompt beforehand
```

Comment:
```suggestion
            "prompt": prompt + " " + lora,  # loras are parsed out of the prompt beforehand
```
minor nitpick, lora/prompt get glued together when displayed in parameters